### PR TITLE
Added lint script and...

### PR DESCRIPTION
**Pull request by bradsk88**
Updated code to pass pylint and flake8

@balloob I know I have push privileges but I'll ask you to look this over since it's sort of a "stylistic change" and I want to make sure at least one other person is on board :]

### CODE REVIEW

**Comment by balloob** [comment]
As we're on this level, you think we should hook it up to travis to have style checks be done on every PR? I did it for [Home Assistant](https://github.com/balloob/home-assistant/) and it is working very well.

As it looks like you've taken an interest in this repo, do you want to take it off my hands ? As stated in the README, I've been looking for a maintainer. I'm happy to transfer the repo to you. Or, if you want, I can transfer it to the [home-assistant github organization](https://github.com/home-assistant) and add you.

**Comment by balloob** [comment]
The changes look good btw, :+1: :dolphin: 

Merge when you see fit.

File: pywink/__init__.py [hunk 1]
```diff
@@ -1,75 +1,80 @@
...
     def name(self):
-        return self.jsonState.get('name', "Unknown Name")
+        return self.json_state.get('name', "Unknown Name")
 
     def state(self):
         raise NotImplementedError("Must implement state")
 
-    def deviceId(self):
+    def device_id(self):
         raise NotImplementedError("Must implement state")
```
Comment (balloob) [inline pywink/__init__.py]: This message needs updating.

**Comment by bradsk88** [comment]
@balloob I'd be happy to take over maintaining the repo.  Thanks.

**Pull request merged**